org.freedesktop.Sdk.Extension.ocaml.yaml

```diff
@@ -0,0 +1,104 @@
+# flatpak-builder --force-clean build-dir org.freedesktop.Sdk.Extension.ocaml.yaml
+# flatpak-builder --user --install --force-clean build-dir org.freedesktop.Sdk.Extension.ocaml.yaml
+id: org.freedesktop.Sdk.Extension.ocaml
+branch: '22.08'
+runtime: org.freedesktop.Sdk
+build-extension: true
+sdk: org.freedesktop.Sdk
+runtime-version: '22.08'
+separate-locales: false
+appstream-compose: false
+build-options:
+  append-path: /usr/lib/sdk/ocaml/bin
+
+modules:
+  - name: bwrap
+    buildsystem: autotools
+    build-options:
+      prefix: ${FLATPAK_DEST}
```

Comment:
the prefix should be set in the top level instead.

Reply:
Moved every prefix to the global top level.